packages/zosfiles/__tests__/__system__/api/methods/upload/Upload.system.test.ts

```diff
@@ -114,6 +114,11 @@ describe("Upload Data Set", () => {
             });
 
             it("should upload a file to a physical sequential data set using Windows relative path", async () => {
+                if (process.platform !== "win32") {
+                    // TODO These tests cannot run on Linux, how should we handle this?
+                    return;
+                }
```

Comment:
There may be a few tricks to work around this.
 See https://github.com/facebook/jest/issues/3652#issuecomment-521359778

Reply:
Ah that is clever :)